libs/src/service-selection/ServiceSelection.js

```diff
@@ -225,15 +244,15 @@ class ServiceSelection {
 
   /**
    * What the criteria is for a healthy service
-   * @param {Response} response axios response
+   * @param {response} response axios response
```

Comment:
I do think `Response` is more correct here because it's an class/object not a primitive, but it's also wrong to write something other than object or w/e. figured i'd point out